manager/src/main/java/com/redhat/service/bridge/manager/actions/connectors/SlackAction.java

```diff
@@ -47,16 +47,9 @@ public JsonNode connectorPayload(BaseAction action) {
 
         ObjectNode objectNode = mapper.createObjectNode();
 
-        ObjectNode connectorMap = mapper.createObjectNode();
-        connectorMap.set("channel", new TextNode(slackChannel));
-        connectorMap.set("webhookUrl", new TextNode(webHookURL));
-
-        objectNode.set("connector", connectorMap);
-
-        ObjectNode kafkaMap = mapper.createObjectNode();
-        kafkaMap.set("topic", new TextNode(kafkaTopic));
-
-        objectNode.set("kafka", kafkaMap);
+        objectNode.set("slack_channel", new TextNode(slackChannel));
```

Comment:
@kostola Can we make these constants. We seem to have a mixture of approaches in the same class e.g. we have 
`public static final String CHANNEL_PARAMETER = "channel";` and then the in-line definitions here.

Reply:
Done :+1: